car-data-app/pages/api/info.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+export default async function handler(req, res) {
+  if (req.method === "GET") {
+    const make = req.query.make || "";
+    const model = req.query.model || "";
+
+    const options = {
+      method: "GET",
+      url: "https://car-data.p.rapidapi.com/cars",
+      params: {
+        limit: "50",
+        page: "0",
+        make: make,
+        model: model,
+      },
+      headers: {
+        "x-rapidapi-host": "car-data.p.rapidapi.com",
+        "x-rapidapi-key": process.env.NEXT_PUBLIC_RAPIDAPI_KEY,
+      },
+    };
+
+    axios
```

Comment:
You can also use `async...await` here.

Reply:
I am using the default code snippet of RapidAPI Hub. I mention this in every application article that RapidAPI Hub provides us with code snippets in different languages. If I change it, it will contradict the snippet that is on RapidAPI Hub.

Reply:
Of course, but I am using the default code snippet of RapidAPI Hub. I mention this in every application article that RapidAPI Hub provides us with code snippets in different languages. If I change it, it will contradict the snippet that is on RapidAPI Hub.